docs/Feature-Broadcasting.md

```diff
@@ -1,12 +1,19 @@
 # Using the Broadcast Feature
-The Player, Heuristic and Internal brains have been updated to support broadcast. The broadcast feature allows you to collect data from your agents in python without controling them.  
+
+The Player, Heuristic and Internal brains have been updated to support broadcast. The broadcast feature allows you to collect data from your agents in using a Python program without controlling them.  
```

Comment:
"agents in using a Python program" read a little funny to me

Reply:
fixed.